content/posts/2022/retention-rules.md

```diff
@@ -0,0 +1,61 @@
+---
+title: "Automatically delete workspace files with retention rules"
+date: 2022-11-29T10:30:00+01:00
```

Comment:
We can already proactively set 2022-12-07 when we shall release this on REANA@CERN.